packages/app/features/Send/PasscodeInput/components/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import { type FC } from 'react';
+import { StyleSheet } from 'react-native';
+import { Button, View } from '@walless/gui';
+import { ChevronLeft } from '@walless/icons';
+
+interface Props {
+	onBack: () => void;
```

Comment:
these kind of props should be optional, `onBack?: () => void;`

Reply:
e.g for mockup phase, we want to display this Component without binding any event-handler yet